zero_bin/.cargo/config.toml

```diff
@@ -2,3 +2,7 @@
 # https://github.com/rust-lang/rust/pull/124129
 # https://github.com/dtolnay/linkme/pull/88
 rustflags = ["-Zlinker-features=-lld"]
+
```

Comment:
`mv ./zero_bin/.cargo/config.toml ./.cargo/config.toml`

Reply:
Hmm yeah I thought about this as well. I'm good with moving this up, but the reason it was placed in `zero-bin` is just because this is the only crate that uses it.